File: src/basic_training/Day19_2.java
package basic_training;

import java.util.ArrayList;

public class Day19_2 {
	public int[] solution(int[] arr) {
		ArrayList<Integer> list = new ArrayList<>();
		for(int i=0; i<arr.length; i++) {
			for(int j=0; j<arr[i]; j++) {
				list.add(arr[i]);
			}
		}
		return list.stream().mapToInt(i -> i).toArray();
	}
}

// .mapToInt(i -> i) : Integer 객체를 int 값으로 변환

File: src/beginer_training/Day19_2.js
function solution(my_str, n) {
  let arr = [];
  for(let i=0; i<my_str.length; i+=n) {
    arr.push(my_str.substring(i, i+n));
  }
  return arr;
}

// 배열.slice()
// 원본배열 영향x, 새로운 배열 반환

// 문자열.substring()
// 원본 문자열 영향x, 잘라진 문자열 반환